### ISSUE TITLE
Insufficient color contrast on some elements

### ISSUE THREAD

**Issue by bbusenius** [uchicago-library/library_website#661]
## LIB20230113-01. Main navigation links insufficient color contrast on focus (Major)
**Issue**: The focus color of main nav dropdown menu links (#487286) has [insufficient contrast](https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum.html) against the gray background color (#434343f2) with a contrast ratio: 1.89:1. The hover color (#EDD4C1) is fine with a contrast ratio: 6.96:1. We recommend updating the CSS to set the focus color to match the hover color.

[Full issue description](https://docs.google.com/document/d/15B0iu5qEQOI8BH-P48pgJul1-8A5b9d6Bi6PwoQvlTc/edit#heading=h.twh4oit61lb7)

## LIB20230113-05. Footer links insufficient color contrast on hover and focus (Major)
**Issue**: Hover and focus The hover and focus color of footer links (#D49464) has [insufficient contrast](https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum.html) against the footer’s gray background color (#494949) with a contrast ratio: 2.55:1. The CSS should be updated to specify a hover and focus color that has a color contrast of at least 4.5:1 with the gray background.

[Full issue description](https://docs.google.com/document/d/15B0iu5qEQOI8BH-P48pgJul1-8A5b9d6Bi6PwoQvlTc/edit#heading=h.6f1iaqva500i)
